src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,26 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import className from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    user,
+    title,
+    completed,
+  } = todo;
+
+  return (
+    <>
+      <article className={className('TodoInfo',
+        { 'TodoInfo--completed': completed === true })}
```

Comment:
This condition would be enough. It works the same way.
```suggestion
        { 'TodoInfo--completed': completed })}
```